data/blog/2022-01-05-serverless-react-native.mdx

```diff
@@ -7,27 +7,27 @@ image: reactnative/cover.jpg
 tags: [serverless, redis, react-native, mobile, aws-lambda]
 ---
 
-In this post, we will use React Native, Serverless framework and Upstash to develop a mobile application for viewing and updating a leaderboard.
+In this post, we will use React Native, the Serverless framework, and Upstash to develop a mobile application for viewing and updating a leaderboard.
 
 We will use React Native to develop the mobile application backed by the Serverless framework, which consists of Python functions running on AWS Lambda.
 
 ### **1 - Using Upstash Redis**
 
-In a typical leaderboard app, user information and scores belonging to users are needed to be stored. Since all of these data should be sorted by scores, using Redis is one of the best possible solutions.
+In a typical leaderboard app, user information and scores belonging to users need to be stored. Since all of this data should be sorted by scores, using Redis is one of the best possible solutions.
 
-“Sorted Set”, which is supported by Redis is a sorted data type that enables users to do storing, adding, removing and querying by range operations in a very fast manner.
+âSorted Set,â which is supported by Redis, is a sorted data type that enables users to perform storing, adding, removing, and querying by range operations in a very fast manner.
```

Comment:
I think there is an issue on here?

Reply:
Thanks for pointing out, I am working on the prompt right now